docs/examples/commands.md

```diff
@@ -0,0 +1,52 @@
+# Slash Commands
+
+In this example, you'll get to know how to create commands and listen to incoming interactions.
+
+## Creating a Command
+
+First off, we need to create a command so that users can use it. We will create an `echo` command which simply returns what the user inputted. Note that global commands can take up to an hour to appear in the client, so if you want to test a new command, you should create it for one guild first.
+
+```js
+// The data for our command
+const commandData = {
+  name: 'echo',
+  description: 'Replies with your input!',
+  options: [{
+    name: 'input',
+    type: 'STRING',
+    description: 'The input which should be echoed back',
+    required: true,
+  }],
+};
+
+client.once('ready', () => {
+  // Creating a global command
+  client.application.commands.create(commandData);
+
+  // Creating a guild-specific command
+  client.guilds.cache.get('id').commands.create(commandData);
+});
+```
+
+And that's it! As soon as your client is ready, it will register the `echo` command.
+
+## Handling Commands
+
+Now let's implement a simple handler for it:
+
+```js
+client.on('interaction', interaction => {
+  // If the interaction isn't a slash command, return
+  if (!interaction.isCommand()) return;
+
+  // Check if it is the correct command
+  if (interaction.commandName === 'echo') {
+    // Get the input of the user
+    const input = interaction.options[0].value;
```

Comment:
This should be `const input = interaction.options.get('input').value`

Reply:
This PR was already merged some time ago, it makes no sense to continue reviewing it.
This line has already been changed by #5705.

Reply:
Thanks for the heads up! I stumbled on this PR from Google and had a heck of time tracking down better documentation since this work hasn't made it to a stable release just yet. I'll check out #5705 and work from there. Thank you monbrey!

Reply:
That file along with other docs were removed in f8703e3e59839d7380795ecf719d5bac691860a4. So, there is nothing to change now, thanks for reporting the issue anyway 👍